scrapers/ArchAngelVideo.yml

```diff
@@ -0,0 +1,28 @@
+name: "Arch Angel Video"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - archangelvideo.com/tour/trailers
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $videoContent: //div[@class="content"][1]
+    scene:
+      Title: $videoContent//h4/a/text()
+      Date:
+        selector: $videoContent[1]//ul[@class="item-meta"]//li[1]/text()
```

Comment:
`$videoContent[1]` could be changed  to `$videoContent` since the `[1]` is already in the `common` declaration of `videoContent`